compliance-report/generate.sh

```diff
@@ -21,12 +21,17 @@ elif [ -n "$SECURITY_REPORT_URL" ]; then
     SECURITY_REPORT="See $SECURITY_REPORT_URL"
 fi
 
+THIRD_PARTY_DEPENDENCY_TRACKER="n/a"
+if [ -n "$THIRD_PARTY_DEPENDENCY_TOOL" ]; then
+    THIRD_PARTY_DEPENDENCY_TRACKER="$THIRD_PARTY_DEPENDENCY_TOOL"
+fi
+
 cat << EOF >> ${S3_ASSETS}/ssdlc_compliance_report.txt
 Release Creator
 ${RELEASE_CREATOR}
 
 Tool used to track third party vulnerabilities
-Silk
+${THIRD_PARTY_DEPENDENCY_TRACKER}
```

Comment:
This is backwards incompatible, can we default to Silk instead of n/a?

Reply:
Actually, I went with Andreas' suggestion (defaulting it in the action.yml).